core/src/main/java/nl/nn/adapterframework/util/MessageKeeperMessage.java

```diff
@@ -74,6 +71,6 @@ public String getMessageText() {
 		return messageText;
 	}
 	public String getMessageLevel() {
-		return messageLevel;
+		return messageLevel.name();
 	}
```

Comment:
Waarom geeft deze niet een MessageKeeperLevel terug?

Reply:
Omdat de getter alleen gebruikt wordt in de console/logging en dat altijd een string is/moet zijn. Enum is vooral van belang bij het zetten van het juiste level.